data/templates/yealink.macros

```diff
@@ -1087,3 +1087,23 @@
     -%}
 {{- map[dtmf] ?? '1' -}}
 {% endmacro dtmf %}
+
+#
+# Backlight timer
+#
+{% macro backlight_time(time) %}
+    {%- set map = {
+        "-1" : "0",
+        "0" : "1",
+        "15" : "15",
+        "30" : "30",
+        "60" : "60",
+        "120" : "120",
+        "300" : "300",
+        "600" : "600",
+        "1800" : "1800"
+        }
+    -%}
+{{- map[time] ?? '' -}}
+{% endmacro backlight_time %}
+
```

Comment:
The macro is not required. See https://nethesis.github.io/tancredi/variables#backlight_time

0 is "always on", -1 is not defined any more. Other values are 1:1

```suggestion
```